packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -37,12 +37,16 @@ class CupertinoButton extends StatefulWidget {
     this.padding,
     this.color,
     this.disabledColor = CupertinoColors.quaternarySystemFill,
-    this.minSize = kMinInteractiveDimensionCupertino,
+    this.minSize,
```

Comment:
looking back at this, I'm a little sad that I made this a `double` rather than a `Size` originally. Though I think to avoid confusing on the API, we might not need to add this here. You can just wrap the CupertinoButton with ConstrainedBox with minWidth and minHeight around the CupertinoButton (and pass to it a minSize of 0) to achieve the same effect via composition.